.github/workflows/e2e.yml

```diff
@@ -35,5 +35,8 @@ jobs:
         with:
           name: artifacts-${{ matrix.os }}
           path: |
-            run/playwright-itest
-            !run/**/SS
+            packages/zui-player/run/playwright-itest
+            !packages/zui-player/run/**/SS
+            packages/zui-player/test-results
+            /var/log/sys*log*
```

Comment:
This particular path is somewhat lazy with the wildcards because it happens to match the system log files on both Linux (`/var/log/syslog*`) and macOS (`/var/log/system.log*`), and this flexible Workflow has the potential to be run on macOS so I figured we might as wel match both.

Now, even as I rationalize this, I recognize I _don't_ currently have a path here to get the equivalent logs on Windows. But in full disclosure, I lack the skills to do much with Windows logs at this point, so I'll cross that bridge when I get to it. 😛